actions/release/create-lang-fam-release-body/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:alpine
```

Comment:
The current action is called ` actions/release/create-lang-fam-release-body`. Can we rename this to `actions/release/notes/language-family`? I would like to refactor the same notes generation for implementations out into an action and having a directory grouping like that would be nice.